src/@context/GlobalData/_types.ts

```diff
@@ -0,0 +1,11 @@
+export interface OpcValue {
```

Comment:
this all is so generic in naming, I wouldn't know what "OceanProtocolCommunityValue" written out would exactly get, and "GlobalData" is kinda what all our context providers hold.

As for file organization, full refactor way would be to merge `useSiteMetadata` hook and this context in this PR into a new e.g. `MarketMetadata` provider, where all values added in here could be put under a `fees` object

Reply:
Agree on all the naming stuff. Added that context like this so that we can fix/evolve in another pr so that this one doesn't get any bigger.